src/Microsoft.ML.Data/Evaluators/MulticlassClassificationEvaluator.cs

```diff
@@ -964,6 +991,15 @@ private IDataView DropPerClassColumn(IDataView input)
             return input;
         }
 
+        private IDataView DropAllTopKColumn(IDataView input)
```

Comment:
I think this method is somewhat unnecessary. This was added on the `MulticlassClassificationMamlEvaluator`. But MAML is a deprecated tool built on top of ML.NET. There's a `DropPercClassColumn` method which drops the per-class LogLoss column, but that was only because there was a `OutputPerClassStatistics` option in MAML, if it was false, the column was dropped.

Since there's no such an option for your new metric array, and since MAML is deprecated, I think it's better to simply not drop the column.

Reply:
IIRC there were tests that failed without this. What is strategy on failing MAML tests if it is deprecated?

Reply:
If that was the case, then I'd recommend keeping this method in order for the tests not to fail.